### PR TITLE
tree-wide: use config_parse_many() and null_or_empty_path()

### PR DESCRIPTION
**Pull request by yuwata**

### CODE REVIEW

File: src/journal/journald-server.c [hunk 1]
```diff
@@ -1625,17 +1625,19 @@ static int server_parse_config_file(Server *s) {
...
                 /* If we are running in namespace mode, load the namespace specific configuration file, and nothing else */
                 namespaced = strjoina(PKGSYSCONFDIR "/journald@", s->namespace, ".conf");
-
-                r = config_parse(NULL,
-                                 namespaced, NULL,
-                                 "Journal\0",
-                                 config_item_perf_lookup, journald_gperf_lookup,
-                                 CONFIG_PARSE_WARN, s,
-                                 NULL);
+                dropin_dirname = strjoina("journald@", s->namespace, ".conf.d");
```
Comment (bluca) [inline src/journal/journald-server.c:1632]: is support for drop-ins already documented?

Comment (bluca) [reply src/journal/journald-server.c:1632]: same question for the link drop-ins

File: src/network/netdev/netdev.c [hunk 2]
```diff
@@ -714,10 +714,6 @@ int netdev_load_one(Manager *manager, const char *filename) {
                 return 0;
         }
 
-        r = fseek(file, 0, SEEK_SET);
```
Comment (keszybz) [inline src/network/netdev/netdev.c:717]: Is `file` used for anything? It looks like creation of `file` could be dropped altogether.

EDIT: oh, it's removed in the next commit. So maybe combine those in one patch?

**Comment by yuwata** [comment]
@bluca and @keszybz Thank you for your review. Now, man pages are updated, and the commit about `fseek()` is merged with the subsequent commit. PTAL.

**Review by bluca — APPROVED** [review]
lgtm

**Pull request merged**